### ISSUE TITLE
aurto does not update packages

### ISSUE THREAD

**Issue by wararjey** [alexheretic/aurto#69]
I just discovered that my aurto installation has old packages in it that do not get updated when the timer runs.

I tried to run /usr/lib/aurto/update-aurto manually, but that did not trigger updates, either.

I am a little bit at loss whats happening, as aurto status does not give me anything useful, there is one log entry i cannot figure out
```
  Feb 10 19:34:42 hostname sudo[145259]: pam_systemd_home(sudo:session): systemd-homed is not available: Unit dbus-org.freedesktop.home1.service not found.
```
Aurto add works normally and updates packages when i re-add any package, though, so I guess my config is not broken...

Regards,

**Comment by wararjey** [comment]
ah, nevermind, aurutils was at fault. Updated aurutils manually and the problem was gone.

Regards,

**Comment by alexheretic** [comment]
Yes this is a bug in aurutils 4.3. I posted a manual fix on the aur page https://aur.archlinux.org/packages/aurto#comment-851163
